src/server/game/DungeonFinding/LFG.h

```diff
@@ -51,6 +51,7 @@ enum LfgUpdateType
     LFG_UPDATETYPE_PROPOSAL_FAILED               = 8,
     LFG_UPDATETYPE_PROPOSAL_DECLINED             = 9,
     LFG_UPDATETYPE_GROUP_FOUND                   = 10,
+    LFG_UPDATETYPE_READDED_TO_QUEUE              = 11,
```

Comment:
I kinda went out on a limb, but it fit perfectly in the enum as if that was what was missing. The code functions without it, though.